components/bigdatacorp/actions/get-address-data/get-address-data.mjs

```diff
@@ -0,0 +1,41 @@
+import app from "../../bigdatacorp.app.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "bigdatacorp-get-address-data",
+  name: "Get Address Data",
+  description: "Returns the available data for a Zipcode number according to the selected dataset. [See the documentation](https://docs.bigdatacorp.com.br/plataforma/reference/enderecos_legal_amazon)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    doc: {
+      propDefinition: [
+        app,
+        "doc",
+      ],
+      description: "Zipcode of the address you want to search for, i.e.: `88048-656`",
+    },
+    dataset: {
+      propDefinition: [
+        app,
+        "dataset",
+      ],
+      options: constants.ADDRESS_DATASETS,
+    },
+  },
+
+  async run({ $ }) {
+    const response = await this.app.getAddressData({
+      $,
+      data: {
+        Datasets: this.dataset,
+        q: `zipcode{${this.doc}}`,

```

Comment:
The [documentation](https://docs.bigdatacorp.com.br/plataforma/reference/enderecos_legal_amazon) shows using square brackets instead of curly brackets here. I suggest testing to see which one is correct before making this change.
```suggestion
        q: `zipcode[${this.doc}]`,
```